_includes/footer.html

```diff
@@ -5,35 +5,19 @@
             <div class="row">
                 <div class="footer-col col-md-4">
                     <h3>Code of Conduct</h3>
-                    <p>All Research Bazaar events respect our <a>Code of Conduct</a>.</p>
+                    <p>All Research Bazaar events respect our <a href="http://melbourne.resbaz.edu.au/post/103597415589/research-bazaar-policies-and-code-of-conduct">Code of Conduct</a>.</p>
                 </div>
                 <div class="footer-col col-md-4">
                     <h3>Around the Web</h3>
                     <ul class="list-inline">
                         <li>
-                            <a href="#" class="btn-social btn-outline"><i class="fa fa-fw fa-facebook"></i></a>
+                            <a href="https://twitter.com/resbaz" class="btn-social btn-outline"><i class="fa fa-fw fa-twitter"></i></a>
                         </li>
                         <li>
-                            <a href="#" class="btn-social btn-outline"><i class="fa fa-fw fa-google-plus"></i></a>
-                        </li>
-                        <li>
-                            <a href="#" class="btn-social btn-outline"><i class="fa fa-fw fa-twitter"></i></a>
+                            <a href="https://www.youtube.com/user/ResBaz" class="btn-social btn-outline"><i class="fa fa-fw fa-youtube"></i></a>
                         </li>
                     </ul>
                 </div>
-                <div class="footer-col col-md-4">
```

Comment:
Let's leave the blog link in for now - I would think about making a global blog or blog planet for ResBaz worldwide, and invite all nodes to participate; the more arenas we can provide for people to feel like they are involved in something big, the better.

I'm uncomfortable with dropping copy information since I'm unable to provide formal legal advice, and don't understand the potential implications of this. If you're unconcerned with IP claims, then I guess it doesn't matter - but I'm a big believer in covering one's butt.